modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+.Prerequisites
+
+* The {cli-manager} is installed.
+* You are using {product-title} 4.17 or later.
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
```

Comment:
Before that I think `CliManager` resource should be created to run cli-manager operand. Am I right @kasturinarra ?

Reply:
@ardaguclu you are right, cli-manager operand needs to be created. @cbippley could you please help add that info ? thanks !!